Dockerfile

```diff
@@ -9,9 +9,19 @@ RUN yum update -y && \
     tar \
     curl \
     wget \
-    net-tools
+    net-tools \
```

Comment:
@alex-ber  you can minimize patches by adding on the line BEFORE the last.

Reply:
Ok.